<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMidyAX</class>
 <widget class="QDialog" name="AboutMidyAX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About MidyAX 2</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Version :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMidyAX">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MidyAX 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersionID">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VersionID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBy">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>By PalyGAP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLeafONICsLOGO">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="CCtoAX.qrc">:/Pictures/pictures/LeafONICs_LOGO.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPALYGAPWebLink">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;http://forum.fractalaudio.com/members/palygap.10408/&quot;&gt;PALYGAP&lt;/a&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewsWebLink">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;https://forum.fractalaudio.com/threads/news-of-the-midyax2-pilot-the-axe-fx-with-midi-controllers-x-touch-bcr2000-guitar-wing.68598/page-6#post-1475734&quot;&gt;News of the MidyAX project&lt;/a&gt;

</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompileDate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compile Date</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="CCtoAX.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutMidyAX</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutMidyAX</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
